src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day calendar__day--1-day"></div>
```

Comment:
You don't need a separate class for each day (`calendar__day--1-day`, `calendar__day--2-day`, etc.), you go through all these divs with the help of a cycles.